plugins/k8saudit-gke/rules/k8s_audit_rules_gke.yaml

```diff
@@ -0,0 +1,100 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2023 The Falco Authors.
```

Comment:
```suggestion
# Copyright (C) 2024 The Falco Authors.
```

Reply:
Please check other file headers, too.

Reply:
updated the copyright text; not sure if github is able to track changes nicely as I am force pushing to rebase on top of the ci fixes branch